README.md

```diff
@@ -2,11 +2,9 @@
 
 # nodemon
 
-For use during development of a node.js based application.
+nodemon is a tool that helps develop node.js based applications by automatically restarting the node application when file changes in the directory are detected. 
 
-nodemon will watch the files in the directory in which nodemon was started, and if any files change, nodemon will automatically restart your node application.
-
-nodemon does **not** require *any* changes to your code or method of development. nodemon wraps your node application and keeps an eye on any files that have changed. Remember that nodemon is a replacement wrapper for `node`, think of it as replacing the word "node" on the command line when you run your script.
+nodemon does **not** require *any* additional changes to your code or method of development. nodemon is a replacement wrapper for `node`, to use `nodemon` simply replace the word `node` on the command line when executing your script.
```

Comment:
Can you remove "simply". It's not required, and it suggests that it's an easy and obvious task, yet anyone actually reading the docs isn't going to have the benefit of what's easy and obvious in nodeland.

Otherwise, happy to merge once it's changed.